examples/Example projects/LedStripDriver_OpenSmart/LedStripDriver_OpenSmart.ino

```diff
@@ -0,0 +1,165 @@
+/*
+   Homey Arduino library
+   Usage example for use with Wemos D1 and Led Strip Driver (open-smart)
+
+   Homey library: https://github.com/athombv/homey-arduino-library
+   Driver for P9813 chip: ChainableRGB: https://github.com/pjpmarques/ChainableLED
+
+   Hardware:
+   - WeMos D1 mini V2
+   - RGB LED Strip Driver Module Shield
+   
+*/
+
+#include <ESP8266WiFi.h>        //ESP8266 library
+#include <WiFiClient.h>         //ESP8266 library
+#include <Homey.h>              //Athom Homey library
+#include <ChainableLED.h>       //P9813 driver library
+
+//just for debugging to serial.print: use DEBUG_PRINT instead of Serial.print
```

Comment:
Please remove these macros, users won't understand how this works. Just keep the Serial.print's in the locations where they are usefull for the user and remove all other prints.